project/web/gunicorn.conf

```diff
@@ -7,9 +7,8 @@ autorestart=true
 stopasgroup=true
 killasgroup=true
 stopwaitsecs=60
-stdout_logfile={{ log_dir }}/gunicorn.log
-redirect_stderr=true
-stderr_logfile={{ log_dir }}/gunicorn.error.log
+stdout_syslog=true
```

Comment:
4.0 isn't released yet, right?  So for now we should probably just set the log file names to `syslog`.  (But it's not mentioned in the 4.0 docs, so we'll have to change to `std(out|err)_syslog` when we upgrade.)